sources/platform/storage/request_queue.md

```diff
@@ -457,23 +467,71 @@ await requestQueueClientOne.batchAddRequests([
 const processingRequestsClientOne = await requestQueueClientOne.listAndLockHead(
     {
         limit: 2,
-        lockSecs: 60,
+        lockSecs: 120,
     },
 );
 
+// Checks when the lock will expire. The locked request will have a lockExpiresAt attribute.
+const theFirstRequestLockedByClientOne = processingRequestsClientOne.items[0];
+const requestLockedByClientOne = await requestQueueClientOne.getRequest(
+    theFirstRequestLockedByClientOne.id,
+);
+console.log(`Request locked until ${requestLockedByClientOne?.lockExpiresAt}`);
+
+// Prolongs the lock of the first request or unlocks it.
+await requestQueueClientOne.prolongRequestLock(
+    theFirstRequestLockedByClientOne.id,
+    { lockSecs: 120 },
+);
+await requestQueueClientOne.deleteRequestLock(
+    theFirstRequestLockedByClientOne.id,
+);
+
+// Cleans up the queue.
+await requestQueueClientOne.delete();
+
+await Actor.exit();
+```
+
+</TabItem>
+<TabItem value="Actor 2" label="Actor 2">
+
+```js
+import { Actor, ApifyClient } from 'apify';
+
+await Actor.init();
+
+const client = new ApifyClient({
+    token: 'MY-APIFY-TOKEN',
+});
+
+// Waits for the first Actor to lock the requests.
+await new Promise((resolve) => setTimeout(resolve, 5000));
+
+// Creates a new request queue.
+const requestQueue = await client.requestQueues().getOrCreate('example-queue');
+
+const requestQueueClientTwo = client.requestQueue(requestQueue.id, {
+    clientKey: 'requestqueuetwo',
+});
+
+// Get all requests from the queue and check one locked by the first Actor.
+const requests = await requestQueueClientTwo.listRequests();
+const requestLockedByClientOne = requests.items.filter((request) => request.lockedByClientKey === 'requestqueueone');
+const theFirstRequestLockedByClientOne = requestLockedByClientOne[0];
+
 // Other clients cannot list and lock these requests; the listAndLockHead call returns other requests from the queue.
 const processingRequestsClientTwo = await requestQueueClientTwo.listAndLockHead(
     {
-        limit: 2,
+        limit: 10,
         lockSecs: 60,
     },
 );
-
-// Checks when the lock will expire. The locked request will have a lockExpiresAt attribute.
-const theFirstRequestLockedByClientOne = processingRequestsClientOne.items[0];
-const requestLockedByClientOne = await requestQueueClientOne.getRequest(
-    theFirstRequestLockedByClientOne.id,
+const wasTheClientTwoLockedSameRequest = !!processingRequestsClientTwo.items.find(
```

Comment:
Here I'm also not sure what the varaible means, could you rename it, or explain better in the comment?